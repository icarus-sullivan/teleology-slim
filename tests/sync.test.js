const { slim } = require('../src');

const context = {
  location: () => 'world',
  what: 'slim',
};

const template = `
hello {{ location }}

this is a test script for {{ what }}`;

test('Template is created', () => {
  const result = slim(template, context);
  expect(result).toMatchSnapshot();
});
