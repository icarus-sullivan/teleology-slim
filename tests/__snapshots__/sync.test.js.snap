// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Template is created 1`] = `
"
hello world

this is a test script for slim"
`;
